languages/ruby/exercises/concept/exceptions/.meta/example.rb

```diff
@@ -0,0 +1,16 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '/', '*']
+
+  UnsupportedOperation = Class.new(StandardError)
+
+  def self.calculate(first_operand, second_operand, operation)
+    raise ArgumentError.new unless (first_operand.is_a?(Number) && second_operand.is_a?(Number))
```

Comment:
```suggestion
    raise ArgumentError unless (first_operand.is_a?(Number) && second_operand.is_a?(Number))
```

Isn't this the syntax that is now "hip"?

Reply:
They are equivalent, I'm not sure which one is more trendy 😸 But your suggestion is shorter so let's go with it.